R/join_key.R

```diff
@@ -65,6 +72,12 @@ join_key <- function(dataset_1, dataset_2 = dataset_1, keys) {
     keys <- NULL
   }
 
+  parents <- if (directed && dataset_1 != dataset_2) {
+    setNames(list(dataset_1), dataset_2)
+  } else {
+    list()
+  }
+
   structure(
     list(
       structure(
```

Comment:
If we are going back to `setNames`, I'm afraid you have to change all instances.

Reply:
I thought about that before accepting the suggestion from @gogonzo, however this doesn't mix the 2 on the _same_ expression.

`setNames` seems fine for such a simple variable construct. 

If you feel strongly about this, I'd prefer reverting to `structure` above.

Reply:
You're right. Sorry, I was reading too fast.